worker-queue-rabbit/src/main/java/com/hpe/caf/worker/queue/rabbit/RabbitWorkerQueueConfiguration.java

```diff
@@ -71,6 +71,12 @@
     @Min(1)
     private int retryLimit;
 
+    /**
+     * The maximum message priority supported by this worker queue. 0 to disable priority feature.
+     */
+    @Min(0)
+    @Max(255)
+    private int maxPriority;
```

Comment:
Is making this an `int` a breaking change?  I'm not 100% how this would serialise is `maxPriority` was not present but we should aim to get it to have the correct default (as we cannot expect that all workers will move to environment variable based configuration immediately).  I would suggest that this should be `Integer` so that it can be `null`, and that the default JavaScript configuration file defaults to `undefined` rather than `255`, and that the code actually handles the default in this case.